app/controllers/additional_details/submissions_controller.rb

```diff
@@ -44,6 +44,8 @@ def create
     @submissions = @protocol.submissions
     @line_item = LineItem.find(submission_params[:line_item_id])
     @service_request = @line_item.service_request
+    @permission_to_edit = @protocol.project_roles.where(identity: current_user, project_rights: ['approve', 'request']).any?
```

Comment:
@jayhardee9 
I originally planned on refactoring this code segment and other uses of it in the Dashboard, but that was a bit much for this PR. This is the way we've done it in the rest of the Dashboard code, so for now, I think keeping the uniformity might be better.